src/core_codemods/harden_pyyaml.py

```diff
@@ -44,6 +47,13 @@ def rule(cls):
 
         """
 
-    def on_result_found(self, _, updated_node):
-        new_args = [*updated_node.args[:1], self.parse_expression("yaml.SafeLoader")]
+    def on_result_found(self, original_node, updated_node):
+        maybe_name = self.get_aliased_prefix_name(original_node, self._module_name)
+        maybe_name = maybe_name or self._module_name
+        if maybe_name and maybe_name == self._module_name:
```

Comment:
`if maybe_name` not necessary since above you do `or` which will make `maybe_name` be truthy